target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-fritzbox-7530.dtsi

```diff
@@ -167,85 +167,6 @@
 	};
 };
 
-&nand {
-	pinctrl-0 = <&nand_pins>;
-	pinctrl-names = "default";
-	status = "okay";
-
-	nand@0 {
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			partition@0 {
```

Comment:
Is the partition layout different between both devices? If it is the same you can also leave it in this file and only change the `nand-ecc-strength` and `nand-ecc-step-size` settings in the device specific dts files.